.github/workflows/choreo.yml

```diff
@@ -16,23 +16,23 @@ jobs:
         include:
           - artifact-name: Windows-x86_64
             os: windows-2022
-            tauri-build-flags: --target x86_64-pc-windows-msvc
+            tauri-build-flags: --target x86_64-pc-windows-msvc -v -- --workspace
```

Comment:
Are the -v flags intentional?

Reply:
-v flags add extra info theres not really a point not having